docs/docsite/rst/administration/awx-manage.rst

```diff
@@ -6,7 +6,7 @@ The *awx-manage* Utility
 .. index:: 
    single: awx-manage
 
-The ``awx-manage`` utility is used to access detailed internal information of AWX. Commands for ``awx-manage`` should run as the ``awx`` or ``root`` user.
+The ``awx-manage`` utility is used to access detailed internal information of AWX. Commands for ``awx-manage`` should only run as the ``awx`` user.
```

Comment:
This is just a nit. I realize it's a backport so a minor change might not be worth the hassle. That said I always try to be careful about the use of the word "only". In this instance I would prefer to say:

`should run as the awx user only` - the position here makes it clear that "only" qualifies the user that runs the command.

`should only run` - on the other hand, the position here seems to qualify the verb "run" - as if to say you should only run the command, not perform some other action.

again this is a minor style thing and the meaning is clear. just thought I'd point that out for reference.

Reply:
Good point!